developer_setup.md

```diff
@@ -156,7 +156,7 @@ git fetch other_user
 ```bash
 bin/setup                  # Installs dependencies, config, prepares database, etc
 bundle exec rake evm:start # Starts the ManageIQ EVM Application in the background
-bundle exec rails s        # Starts the application server
+bundle exec rails s        # Starts the application server - no need to run this if evm:start was executed 
```

Comment:
@chrisarcand @Fryguy I will remove this line. the PR is initiated due to a fact that developers who are new to miq and new to rails run both evm:start and rails s, because this is how it sounds from the guide.
@kbrock at least outside of miq core team, from my experience a lot of devs use evm:start.
why? I guess because that's what the developer guide says :)
and also how would you test otherwise an entire flow with refresh/events and etc. when developing a provider?